addons/website_sale/data/product_snippet_template_data.xml

```diff
@@ -28,7 +28,8 @@
                         <div>
                             <t t-call="website_sale.price_dynamic_filter_template_product_product"/>
                         </div>
-                        <button type="button" role="button" class="btn btn-primary js_add_cart ml-auto" title="Add to Cart">
+                        <button t-if="record._website_show_quick_add()"
+                                type="button" role="button" class="btn btn-primary js_add_cart ml-auto" title="Add to Cart">
```

Comment:
Hello @rdeodoo @bso-odoo,

If this change is validated, I'll have to write an upgrade script for this, I guess. Is it possible to give some insight regarding what should be upgraded ? (I read a bit the upgrade script of searchbar snippet and product recently viewed)

Thanks in advance

Reply:
Here is some insight: those templates having `id` starting with `dynamic_filter_template_` are actually not saved in existing pages and therefore do not have to be migrated within pages in the way snippets sometimes need to (like what we did for searchbar).
They are actually rendered on the server by `website.snippet.filter`'s `_render()` which is accessed through the `/website/snippet/filters` route.
Unfortunately they are defined in a `<data noupdate="1">` block.
Therefore, last time this discussion happened, it was decided to remove this `noupdate` and to migrate past ones to remove that flag too: https://github.com/odoo/upgrade/pull/3169. I believe the same migration would apply here.

Reply:
Okay ! Thanks a lot !